high_card.rb

```diff
@@ -1,5 +1,52 @@
-# Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
-suits = [ "hearts", "spades", "clubs", "diamonds" ]
+require "pry"
 
-players = []
+def build_deck
+  ranks = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A" ]
+  suits = [ "hearts", "spades", "clubs", "diamonds" ]
+  deck = []
+  ranks.each_with_index do |rank, i|
+    suits.each do |suit|
+      deck << [rank, suit, i]
```

Comment:
this works, but it might make it a little easier to reference these values later if you make each card a `hash` rather than array, structured like: 

```ruby
deck << {rank: ranks[i], suit: suit, worth: i + 1}
```